CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.14)
```

Comment:
Is there a specific reason for requiring at least 3.14?

Reply:
changed to 3.10, because `gtest_discover_tests()` was introduced in CMake 3.10